src/print.rs

```diff
@@ -389,6 +433,28 @@ fn should_add_implicit_custom_module(
         .unwrap_or(false)
 }
 
+fn should_add_implicit_env_var_module(
```

Comment:
As you might have noticed, there is a bit of code duplication from custom, should another module with this behavior be added, we might want to consider refactoring, or maybe even refactor some/all of it now, though it was complicated enough getting it to work as it is.

Reply:
Shouldn't it be possible to make this particular function work for both by using `let explicit_module_name = format!("{parent_module}.{child_module}");`?